sklearn/preprocessing/_encoders.py

```diff
@@ -145,10 +145,11 @@ class OneHotEncoder(_BaseEncoder):
 
     Parameters
     ----------
-    categories : 'auto' or a list of lists/arrays of values.
+    categories : 'auto' (default) or a list of lists/arrays of values.
         Categories (unique values) per feature:
 
-        - 'auto' : Determine categories automatically from the training data.
+        - 'auto' (default): Determine categories automatically from the
```

Comment:
I think if you tell above that the 'auto' is default, I don't believe that here is necessary.